osu.Game/Online/Chat/ChannelManager.cs

```diff
@@ -445,6 +445,14 @@ protected override Task Poll()
             return tcs.Task;
         }
 
+        public void MarkChannelAsRead(Message message)
```

Comment:
how have you tested this? it's not called from anywhere.

Reply:
I added this so it could be used later on, if you want I could implement this into ChatOverlay to actually set channels as read.

Reply:
It's very hard to merge something if the thing being added has no way of testing it. And it shows that the code has probably been written without a single check as to whether it works.

Reply:
Can you suggest me what type of test to implement? This is API code so I'm unsure how to test such.

Reply:
API testing is still somewhere we are lacking. It's best you implement usage of the function in the same PR as adding it. Feel free to split this into its own PR if that makes more sense.

Reply:
See commit https://github.com/ppy/osu/pull/7425/commits/7cc388b5abc56b5d49d9fabe9596db1701b22641

Reply:
It has also has been tested successfully with kind @mouzedrift
https://streamable.com/qapuj